packages/frontend/src/pages/api/auth/[auth0].ts

```diff
@@ -0,0 +1,3 @@
+import { handleAuth } from "@auth0/nextjs-auth0";
+
```

Comment:
I will do some modification on this file, as on the `Pages Router` of NextJS and deploying on `Vercel`, we need to work around the different deployment URL that Vercel sets as `AUTH0` does not respect `VERCEL_URL` for their `base_url` prop

Reply:
FIxed on commit `ad06446372f54ed6282bf5edadd6ce06ca805d25`